src/main/java/ita/softserve/course_evaluation_analytics/service/CourseSatisfactionService.java

```diff
@@ -0,0 +1,73 @@
+package ita.softserve.course_evaluation_analytics.service;
+
+import ita.softserve.course_evaluation_analytics.entity.Course;
+import ita.softserve.course_evaluation_analytics.entity.CourseFeedbackSatisfaction;
+import ita.softserve.course_evaluation_analytics.entity.FeedbackRequest;
+import ita.softserve.course_evaluation_analytics.repository.AnalystRepository;
+import org.springframework.stereotype.Service;
+
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+
+@Service
+
+public class CourseSatisfactionService {
+
+    AnalystRepository analystRepository;
+
+    CourseSatisfactionService(AnalystRepository analystRepository) {
+        this.analystRepository = analystRepository;
+    }
+
+    List<CourseFeedbackSatisfaction> coursesFeedbackSatisfaction=new ArrayList<CourseFeedbackSatisfaction>();
```

Comment:
what is the reason for creating this variable at class level?

Reply:
moved into method body